src/util/misc.ts

```diff
@@ -173,25 +173,25 @@ export function saveStaticFiles(config: Config, zimCreator: ZimCreator) {
   ]);
 }
 
-export function cssPath({ output: { dirs } }: Config, css: string) {
-  return [dirs.style, `${dirs.styleModules}/${css.replace(/(\.css)?$/, '')}.css`].join('/');
+export function cssPath(css: string, subDirectory = '') {
+  return `${subDirectory ? `${subDirectory}/` : ''}${css.replace(/(\.css)?$/, '')}.css`;
```

Comment:
Do we still need to check `subDirectory` now that we have a default value?

Reply:
it's needed as we are passing just the subdirectory name we need to add `/` between them that's why I used this method